src/clusterfuzz/stacktraces/__init__.py

```diff
@@ -416,6 +416,15 @@ def parse(self, stacktrace: str) -> CrashInfo:
         if state.crash_type in IGNORE_CRASH_TYPES_FOR_ABRT_BREAKPOINT_AND_ILLS:
           continue
 
+      # Windows formats and surfaces ILL differently from Linux/Posix
+      # Update crash type for consistency.
+      self.update_state_on_match(
```

Comment:
Can you please add a test? 

See https://github.com/google/clusterfuzz/commit/da388a2b7e153c05fc8d4fa93aaf411b107c99f3 for an example.

Reply:
yessss thank you for the example! super helpful!

Reply:
(also I overwrote the old commits with the good intention of consolidating/cleaning the commits.. unsure if thats a git taboo - let me know if it is!! wont do it again c: )

Reply:
No need to overwrite older commits. When we merge everything will get squashed in the end :)